packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -85,7 +87,7 @@ export class SlackService {
 
     private integrationKey = process.env['NANGO_SLACK_INTEGRATION_KEY'] || 'slack';
     private nangoAdminUUID = process.env['NANGO_ADMIN_UUID'];
-    private env = 'prod';
+    private env = process.env['NANGO_ADMIN_PROD_ENV'] || 'prod';
```

Comment:
nb in shared you wont be able to use parsed, but you can refactor the class to accept the env in the constructor